.env.docker.example

```
@@ -0,0 +1,34 @@
+
+# ユーザーインポート時のデフォルトパスワード
+DEFAULT_USER_PASSWORD=P@ssw0rd
+
+# pgAdmin
+PGADMIN_DEFAULT_EMAIL=admin@example.com
+PGADMIN_DEFAULT_PASSWORD=admin
+
+# ポート設定
+NGINX_PORT=80
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)